### PR TITLE
add init services

### PR DESCRIPTION
**Pull request by zaibon**
- haveged: random generator
- cgroup creation

### CODE REVIEW

File: config/etc/zinit/haveged.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+exec: haveged -w 1024 -d 32 -i 32 -v 1
+oneshot: true
```
Comment (muhamadazmy) [inline config/etc/zinit/haveged.yaml]: This should be removed, it's daemon so remove oneshot to get restarted if crached

Comment (zaibon) [reply config/etc/zinit/haveged.yaml]: I'm not sure, I didn't find a way to make not make it fork and stay in the foreground

**Comment by muhamadazmy** [comment]
I see. In that case leave it like this until we add support for forked
services.

File: config/etc/zinit/udev-trigger.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-exec: udevadm trigger --action=add; udevadm settle
+exec: /etc/zinit/init/udev.sh
```
Comment (maxux) [inline config/etc/zinit/udev-trigger.yaml]: Add `sh` here or drop it on the others, to be consistant

**Pull request merged**